src/hooks/useSubaccount.tsx

```diff
@@ -275,8 +273,8 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
   const dydxAddress = localDydxWallet?.address as DydxAddress;
 
   useEffect(() => {
-    dispatch(setSubaccount(undefined));
-    dispatch(setHistoricalPnl([]));
+    dispatch(clearSubaccountState());
+    dispatch(clearLocalOrders());
```

Comment:
wdyt about just storing localOrders indexed by account address instead of clearing state when the account changes?

Reply:
mm as in storing them to localStorage? i think we should be ok keeping these as single session/ losing them when disconnecting wallet -- abacus already handles emitting fills related notifications, so the local orders state here are mostly just for the 2-5 seconds of each order placement/cancelation